package.json

```diff
@@ -52,7 +53,6 @@
     "lint-staged": "^15.2.7",
     "postcss": "^8.4.38",
     "prettier": "^3.3.3",
-    "rimraf": "^6.0.1",
```

Comment:
There are tons of scripts that use `rimraf` in other packages as well (`pnpm clean`)

Reply:
Yeah, but it's only using in scripts, not in files directly.
![image](https://github.com/user-attachments/assets/d8858a53-fab5-4a54-a441-e8314e527191)

This global install ```"postinstall": "pnpm i rimraf -g"``` solving this issue.